GameServerLib/Maps/Map.cs

```diff
@@ -84,40 +111,461 @@ public void Update(float diff)
+                    NexusList.Add(new Nexus(_game, MapScript.NexusModels[teamId], teamId, nexusRadius, position, sightRange, Crc32Algorithm.Compute(Encoding.UTF8.GetBytes(mapObject.Name)) | 0xFF000000));
+                }
+                // Inhibitors
+                else if (objectType == GameObjectTypes.ObjAnimated_BarracksDampener)
+                {
+                    //Inhibitor model changes dont seem to take effect in-game
+                    InhibitorList[teamId][lane].Add(new Inhibitor(_game, MapScript.InhibitorModels[teamId], lane, teamId, inhibRadius, position, sightRange, Crc32Algorithm.Compute(Encoding.UTF8.GetBytes(mapObject.Name)) | 0xFF000000));
+                    test[teamId].Add(mapObject);
+                }
+                // Turrets
+                else if (objectType == GameObjectTypes.ObjAIBase_Turret)
+                {
+                    if (mapObject.Name.Contains("Shrine"))
+                    {
+                        TurretList[teamId][lane].Add(new LaneTurret(_game, mapObject.Name + "_A", MapScript.TowerModels[teamId][TurretType.FOUNTAIN_TURRET], position, teamId, TurretType.FOUNTAIN_TURRET, GetTurretItems(TurretType.FOUNTAIN_TURRET), 0, LaneID.NONE, mapObject));
+                        continue;
+                    }
+
+                    int index = mapObject.ParseIndex();
+
+                    var turretType = MapScript.GetTurretType(index, lane, teamId);
+
+                    if (turretType == TurretType.FOUNTAIN_TURRET)
+                    {
+                        missedTurrets.Add(mapObject);
+                        continue;
+                    }
+
+                    if (mapObject.Name.Contains("_Point"))
+                    {
+                        CapturePointsList.Add(new Tuple<IMinion, uint>(new Minion(_game, null, new Vector2(mapObject.CentralPoint.X, mapObject.CentralPoint.Z), "OdinNeutralGuardian", "OdinNeutralGuardian"), _game.NetworkIdManager.GetNewNetId()));
+                        continue;
+                    }
+                    TurretList[teamId][lane].Add(new LaneTurret(_game, mapObject.Name + "_A", MapScript.TowerModels[teamId][turretType], position, teamId, turretType, GetTurretItems(turretType), 0, lane, mapObject));
+                }
+                else if (objectType == GameObjectTypes.ObjBuilding_SpawnPoint)
+                {
+                    AddFountain(teamId, position);
+                }
+                else if (objectType == GameObjectTypes.ObjBuilding_NavPoint)
+                {
+                    BlueMinionPathing[lane].Add(new Vector2(mapObject.CentralPoint.X, mapObject.CentralPoint.Z));
+                }
+            }
+
+            //If the map doesn't have any Minion pathing file but the map script has Minion pathing hardcoded
+            if (BlueMinionPathing.Count == 0 && MapScript.MinionPaths != null && MapScript.MinionPaths.Count != 0 || MapScript.MinionPathingOverride)
```

Comment:
Change this to
```suggestion
            if ((BlueMinionPathing.Count == 0 || MapScript.MinionPathingOverride) && MapScript.MinionPaths != null && MapScript.MinionPaths.Count != 0)
```

Reply:
My intention for the override was that it would actually override everything, .
So if someone wants to do a custom path, they could do even if there were pathing files

Reply:
Conditionals are handled left to right in pairs of two, meaning the original would be `MapScript.MinionPaths.Count != 0 || MapScript.MinionPathingOverride`, the `||` would not be applied to the rest of the conditions before it.